lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -2158,7 +2158,7 @@ export class UserAgentApplication {
      * @returns {boolean} true/false
      */
     public getLoginInProgress(): boolean {
-        return this.cacheStorage.getItem(TemporaryCacheKeys.INTERACTION_STATUS) === Constants.inProgress;
+        return this.cacheStorage.isInteractionInProgress(true);
```

Comment:
Can this be param less `isInteractionInProgress()`?

Reply:
No, in some scenarios we need to know if **any** interaction is in progress and in others we need to know if **this clientId** has interaction in progress. This flag denotes which we care about.

Reply:
Okay. The name confused me a little. Makes sense now.